src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -290,3 +290,15 @@ function _formatLabel(meta: ProfileMeta): string {
   }
   return product + (version ? ` (${version})` : '') + (os ? ` ${os}` : '');
 }
+
+function _formatInterval(interval: number): string {
+  // Format in the most precise base (milliseconds, microseconds, or
+  // nanoseconds), to avoid cases where intervals are displayed with
+  // too many leading zeroes to be useful.
+  if (interval >= 1.0) {
```

Comment:
nit: I can see you're coming from a non-JS background :-) In JS I think it's not needed to force a float comparison using a float as the right term.

Reply:
Hah, you're not wrong.  I switched to using "integers".